.github/actions/bump-submodule/action.yml

```diff
@@ -13,24 +13,25 @@ inputs:
 
 runs:
   using: 'composite'
-  # XXX: Environment variables to suppress git config warning.
-  # The commit is required to avoid resetting dirty
-  # changes in submodule.
-  env:
-    GIT_COMMITTER_NAME: "integration testing workflow"
-    GIT_COMMITTER_EMAIL: "noreply@tarantool.org"
-    GIT_AUTHOR_NAME: "integration testing workflow"
-    GIT_AUTHOR_EMAIL: "noreply@tarantool.org"
   steps:
-    run: |
-      pushd third_party/${{ inputs.submodule }}
-      git fetch origin
-      git checkout ${{ inputs.revision }}
-      popd
-      # XXX: --allow-empty is required to rerun flaky tests
-      # for HEAD when it is already bumped
-      # in tarantool/tarantool repo to the same revision.
-      # Otherwise the command below fails with "nothing to
-      # commit" reason.
-      git commit --allow-empty -m ${{ inputs.submodule }}": integration testing bump" \
-        third_party/${{ inputs.submodule }}
+    - name: Bump ${{ inputs.submodule }}
```

Comment:
Please add the 'submodule' word: `Bump ${{ inputs.submodule }} submodule`.

Also, please update the description of that action. For example, `Bump specified third-party submodule for integration testing`.

Reply:
Fixed, thanks!